src/averageAges.js

```diff
@@ -1,61 +1,69 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateAverageAge(people, filterFunction) {
+  const filteredPeople
```

Comment:
**[severity 3]: Variable declaration and assignment spanning across multiple lines**
            Reformat the code to keep the variable declaration and assignment on the same line
            **How to fix:**
            Move the assignment to a single line